Builder/ListBuilder.php

```diff
@@ -164,7 +164,7 @@ public function buildActionFieldDescription(FieldDescriptionInterface $fieldDesc
         }
 
         if (null === $fieldDescription->getType()) {
-            $fieldDescription->setType('action');
+            $fieldDescription->setType('actions');
```

Comment:
Oh right… I'm not sure what this is used for ? Maybe for deciding which template to use? @andrewmy do you know what the implications of changing this are?